test/programmatic_model_creation/programmatic_model_expansion.jl

```diff
@@ -1,34 +0,0 @@
-#! format: off
```

Comment:
The empty test network test was moved to the reaction system structure test. This file is pretty much deprecated when we removed mutating functions, and hence removed. 

It feels silly and bad that we end up with a folder with only a single file (the composing model test). However, I'd like to test more for hierarchical/composed systems, so we will likely add more test files here? In which case it would make sense to keep this one.